ekip/everykid/templates/base.html

```diff
@@ -1,18 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
 
-{% block head%}
 <head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="HandheldFriendly" content="True">
+  <meta name="MobileOptimized" content="320">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+
+  <link rel="stylesheet" href="/static/css/style.min.css" />
```

Comment:
Also let's use the static template tag, instead of hard-coding these URLs. When we move to using S3 to store our static assets, this will change completely.